src/svcutilcore/src/Resources/Strings.resx

```diff
@@ -169,7 +169,7 @@
     <value>Description: {0} can generate code for service contracts, clients and data types from metadata documents. These metadata documents can be on disk or retrieved online. Online retrieval follows either the WS-Metadata Exchange protocol or the DISCO protocol.</value>
   </data>
   <data name="HelpCodeGenerationSyntax" xml:space="preserve">
-    <value>Syntax: {0} [/{1}:{2}]  {3}* | {4}* | {5}</value>
+    <value>Syntax: {0} [-{1}:{2}]  {3}* | {4}* | {5}</value>
```

Comment:
Should "-" be "--" instead?

Reply:
@Lxiamail This is called in ToolConsole.cs line 205, {1} is Options.Abbr.Target, which is the abbreviation of "target", so we should use "-" here.

Reply:
Could you rename the string resource to make it clearer? Maybe HelpCodeGenerationAbbreviationSyntax?

---
In reply to: [236860575](https://github.com/dotnet/wcf/pull/3274#discussion_r236860575) [](ancestors = 236860575)